app/components/forms/wizard/basic-details-step.js

```diff
@@ -436,7 +436,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
 
     openTaxModal(isNewTax) {
       if (!this.isCreate && isNewTax) {
-        this.set('data.event.tax', this.store.createRecord('tax'));
+        this.set('data.event.tax', this.store.createRecord('tax', { name: this.data.event.tax.name }));
```

Comment:
Why ?

Reply:
the tax data was neot showing correctly only when the case was like
`if(!isCreate && isNewTax)` , this means that when creating the tax info, while it was not created at the time of creating the event. I read from the docs https://guides.emberjs.com/release/models/creating-updating-and-deleting-records/ and in the same file - 
```
this.set('data.event.stripeAuthorization', this.store.createRecord('stripe-authorization', {
            stripeAuthCode       : authorization.authorizationCode,
            stripePublishableKey : ENV.environment === 'development' || ENV.environment === 'test' ? this.settings.stripeTestPublishableKey : this.settings.stripePublishableKey
          }));
```

I was trying to resolve this from 3 days continuosly tried and tried , then tried the above and it worked finally :100:

Reply:
intially i have write it like -
```

 this.set('data.event.tax', this.store.createRecord('tax',
{name: this.data.event.tax.name,
taxId: this.data.event.tax.taxId,
rate: this.data.event.tax.rate,
isTaxIncludedInPrice: this.data.event.tax.isTaxIncludedInPrice,
}));
```
then optimize it to - 
`this.set('data.event.tax', this.store.createRecord('tax',{name: this.data.event.tax.name}));
`
and working of both is same and correct